### PR TITLE
Update code samples

### PR DESCRIPTION
**Pull request by bidoubiwa**
See https://github.com/meilisearch/integration-guides/issues/185
fixes: #145

### CODE REVIEW

File: .code-samples.meilisearch.yaml [hunk 1]
```diff
@@ -8,7 +8,7 @@ get_one_index_1: |-
 list_all_indexes_1: |-
   await client.getIndexes();
 create_an_index_1: |-
-  await client.createIndex('movies', primaryKey: 'movie_id');
+  await client.createIndex('movies');
```
[the remark this answers is not in the document]
Comment (alallema) [reply .code-samples.meilisearch.yaml:11]: Also I check the [`.code-samples.meilisearch.yaml`](https://github.com/meilisearch/documentation/blob/1d1ad5020ac571080b6c03dfc23c0d18fdfc47d1/.code-samples.meilisearch.yaml#L12) from the documentation and they had:
create_an_index_1: |-
```bash
  curl \
    -X POST 'http://localhost:7700/indexes' \
    -H 'Content-Type: application/json' \
    --data-binary '{
      "uid": "movies",
      "primaryKey": "id"
    }'
```
And I let:
`"primaryKey": "movie_id"` everywhere ...

**Review by alallema — COMMENTED** [review]
Just a concern about `create_index`

File: .code-samples.meilisearch.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (bidoubiwa) [reply .code-samples.meilisearch.yaml:11]: So here it is not correct: https://github.com/meilisearch/meilisearch-ruby/pull/308/files ?

[the remark this answers is not in the document]
Comment (bidoubiwa) [reply .code-samples.meilisearch.yaml:11]:
```suggestion
  await client.createIndex('movies',  primaryKey: 'id');
```

[the remark this answers is not in the document]
Comment (brunoocasali) [reply .code-samples.meilisearch.yaml]: Yeah, I think is not correct :)
```yml
document_guide_create_index_primary_key: |-
  curl \
    -X POST 'http://localhost:7700/indexes' \
    -H 'Content-Type: application/json' \
    --data-binary '{
      "uid": "movies",
      "primaryKey": "reference_number"
    }'
    
create_an_index_1: |-
  curl \
    -X POST 'http://localhost:7700/indexes' \
    -H 'Content-Type: application/json' \
    --data-binary '{
      "uid": "movies",
      "primaryKey": "id"
    }'    
```

File: .code-samples.meilisearch.yaml [hunk 2]
```diff
@@ -285,6 +281,59 @@ getting_started_search_md: |-
...
+    'rank:desc'
+  ])
+getting_started_update_searchableAttributes: |-
+  await client.index('movies').updateSearchableAttributes([
+    'title'
+  ])
+getting_started_update_stop_words: |-
+  await client.index('movies').updateStopWords(['the'])
+getting_started_check_task_status_md: |-
+  ```dart
```
Comment (brunoocasali) [inline .code-samples.meilisearch.yaml:302]: Should we keep this _md since they are not using it anymore?

File: .code-samples.meilisearch.yaml [hunk 3]
```diff
@@ -416,3 +460,13 @@ tenant_token_guide_generate_sdk_1: |-
 tenant_token_guide_search_sdk_1: |-
   final frontEndClient = MeiliSearchClient('http://127.0.0.1:7700', token);
   await frontEndClient.index('patient_medical_records').search('blood test');
+landing_getting_started_1: |-
+  var client = MeiliSearchClient('http://127.0.0.1:7700', 'masterKey')
+  await client.index('movies').addDocuments([
```
Comment (brunoocasali) [inline .code-samples.meilisearch.yaml:465]:
```suggestion

  await client.index('movies').addDocuments([
```

**Review by brunoocasali — APPROVED** [review]
🥇

**Comment by brunoocasali** [comment]
bors merge

**Pull request merged**